packages/dd-trace/src/platform/node/profiler.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const log = require('../../log')
+const { profiler, AgentExporter } = require('../../profiling')
+
+let cached
+
+module.exports = config => cached || (cached = {
+  start: () => {
+    const { service, version, env, url, hostname, port } = config
+    const { enabled } = config.profiling
+    const exporter = new AgentExporter({ url, hostname, port })
+    const logger = {
+      debug: (message) => log.debug(message),
+      warn: (message) => log.debug(message),
```

Comment:
Shouldn't warn map to warn? It looks like like warnings will only appear at debug level?

Reply:
`warn` didn't exist on the tracer when this PR was opened, but now that it's there that should indeed be the case.

Reply:
Updated with the correct methods and will be in the next version.